test/k8sT/Services.go

```diff
@@ -412,6 +412,93 @@ var _ = Describe("K8sServicesTest", func() {
 		})
 	})
 
+	Context("Checks local redirect policy", func() {
```

Comment:
LRP depends on the host-reachable services feature which is enabled only on `net-next` and `4.19` CI runs. To skip the suite on other CI runs of other types, you can use `SkipContextIf(func () bool { return !helpers.RunsOnNetNextOr419Kernel() }, "Checks local redirect policy", func () { ...`.

Reply:
Ah makes sense, thanks!